DROD/GameScreen.cpp

```diff
@@ -6069,6 +6096,26 @@ void CGameScreen::ShowLockIcon(const bool bShow)
 	}
 }
 
+//*****************************************************************************
+void CGameScreen::ShowChatHistory(CEntranceSelectDialogWidget* pBox)
+{
+	g_pTheSound->PauseSounds();
+
+	const Uint32 dwSpeechRemaining = this->dwNextSpeech - SDL_GetTicks();
```

Comment:
I note here that if dwNextSpeech == 0, then this unsigned value will translate the negative ts to a large positive.

That seems benign, though, as the value won't be used in this case.